spi/control-plane/contract-spi/src/main/java/org/eclipse/edc/connector/contract/spi/types/offer/ContractOffer.java

```diff
@@ -192,12 +192,11 @@ public Builder policy(Policy policy) {
             return this;
         }
 
-
         public ContractOffer build() {
             Objects.requireNonNull(contractOffer.id);
             Objects.requireNonNull(contractOffer.asset, "Asset must not be null");
             Objects.requireNonNull(contractOffer.policy, "Policy must not be null");
-
+            Objects.requireNonNull(contractOffer.contractEnd, "ContractEnd must not be null");
```

Comment:
A contract with a `contractEnd`, but no `contractStart` seems ... odd. Both should be enforced IMO.